cogs/coding.py

```diff
@@ -297,63 +353,93 @@ async def man(self, ctx: Context, *, page: str) -> None:
         async with ctx.typing():
             async with self.session.get(url) as response:
                 if response.status != 200:
-                    await ctx.send("An error occurred (status code: {response.status}). Retry later.")
+                    await ctx.send(
+                        "An error occurred (status code: {response.status}). Retry later."
+                    )
                     return
 
                     soup = BeautifulSoup(await response.text(), "lxml")
 
                     nameTag = soup.find("h2", string="NAME\n")
 
                     if not nameTag:
-                        return await ctx.send(f"No manual entry for `{page}`. (Debian)")
+                        return await ctx.send(
+                            f"No manual entry for `{page}`. (Debian)"
+                        )
 
-                    contents = soup.find_all("nav", limit=2)[1].find_all("li", limit=3)[1:]
+                    contents = soup.find_all("nav", limit=2)[1].find_all(
+                        "li", limit=3
+                    )[1:]
 
                     if contents[-1].string == "COMMENTS":
                         contents.remove(-1)
 
                     title = self.get_h2_content(nameTag)
 
-                    emb = discord.Embed(title=title, url=f"https://man.cx/{page}")
+                    emb = discord.Embed(
+                        title=title, url=f"https://man.cx/{page}"
+                    )
                     emb.set_author(name="Linux man pages")
-                    emb.set_thumbnail(url="https://www.debian.org/logos/openlogo-nd-100.png")
+                    emb.set_thumbnail(
+                        url="https://www.debian.org/logos/openlogo-nd-100.png"
+                    )
 
                     for tag in contents:
-                        h2 = tuple(soup.find(attrs={"name": tuple(tag.children)[0].get("href")[1:]}).parents)[0]
-                        emb.add_field(name=tag.string.strip(), value="\n".join([i for i in self.get_h2_content(h2).split("\n") if i]), inline=False)
+                        h2 = tuple(
+                            soup.find(
+                                attrs={
+                                    "name": tuple(tag.children)[0].get("href")[
+                                        1:
+                                    ]
+                                }
+                            ).parents
+                        )[0]
+                        emb.add_field(
+                            name=tag.string.strip(),
+                            value="\n".join(
+                                [
+                                    i
+                                    for i in self.get_h2_content(h2).split(
+                                        "\n"
+                                    )
+                                    if i
+                                ]
+                            ),
```

Comment:
This is unacceptable, this can't be split like a list, it's a relatively simple list comprehension and splitting it doesn't make sense